spec/integration/create_subscriber_list_spec.rb

```diff
@@ -93,6 +97,19 @@
       expect(SubscriberList.first.url).to eq("/oil-and-gas")
     end
 
+    it "create a subscriber_list with a digest of the JSON content" do
+      create_subscriber_list(tags: { topics: { any: ["oil-and-gas/licensing"] },
+                                     location: { all: %w[france germany] } },
+                             links: { topics: { any: ["oil-and-gas/licensing"] },
+                                     location: { all: %w[france germany] } })
+      expect(SubscriberList.last.tags_digest).to eq("0cac2ccdb5607c66f8c0ad62cc29e9bdcc09a0b349191406f9db743dac871d4c")
+      expect(SubscriberList.last.links_digest).to eq('0cac2ccdb5607c66f8c0ad62cc29e9bdcc09a0b349191406f9db743dac871d4c')
+
+      create_subscriber_list(tags: { topics: { any: ["oil-and-gas/licensing"] } })
+      expect(SubscriberList.last.tags_digest).to eq("06982c6eaa16c38056dcf81da2bc7ccb7c58e6ef68349db9012534d025a7dc5a")
+      expect(SubscriberList.last.links_digest).to be_nil
```

Comment:
I'd avoid hard coding these digests in here since they make this test rather brittle to change. I think it'd be sufficient that these expectations were just that they matched the output of a call to the digest generator call.

It feels more the role of the digest generator to test that it's returning the right digests.